### ISSUE TITLE
Caching g++ dependencies.

### ISSUE THREAD

**Issue by awalsh128** [awalsh128/cache-apt-pkgs-action#16]
In PR #11 it was mentioned by @jroddev that they were having trouble with g++ dependencies.

> This fixed the "Package 'g++-11' not found." error in the cache action. However caching the newer g++ has proven difficult with all of its dependencies and I haven't been able to get it to work on the 2nd run.

@jroddev could you provide more details of the problem or perhaps the error scenario?

**Comment by jroddev** [comment]
Hey @awalsh128. The escape character issue is fixed with this change https://github.com/awalsh128/cache-apt-pkgs-action/pull/10/files 

However I wasn't able to successfully cache g++-11 because it transitively installs quite a few system files. I tried caching extra packages with no luck here - https://github.com/jroddev/game-frameworks/pull/6/commits/93138d73bf0316e71a37b62260e541acf84cfac1
Most of it seems related to missing cpp std-lib.

I have chosen to just install g++-11 each time now. Looks like it takes about 40-60 seconds.

**Comment by awalsh128** [comment]
Hey @jroddev, I am wondering if this is related to #25 at all?

I'd like to still solve this issue if possible. Could you get the results of `dpkg -L g++-11` for me? Also noting which files are missing.

**Comment by jroddev** [comment]
Build: https://github.com/jroddev/game-frameworks/runs/7369032415?check_suite_focus=true
PR: https://github.com/jroddev/game-frameworks/pull/6
```
---------- RUNNING DPKG ----------
/.
/usr
/usr/bin
/usr/bin/x86_64-linux-gnu-g++-11
/usr/lib
/usr/lib/gcc
/usr/lib/gcc/x86_64-linux-gnu
/usr/lib/gcc/x86_64-linux-gnu/11
/usr/lib/gcc/x86_64-linux-gnu/11/cc1plus
/usr/lib/gcc/x86_64-linux-gnu/11/g++-mapper-server
/usr/share
/usr/share/doc
/usr/share/doc/gcc-11-base
/usr/share/doc/gcc-11-base/C++
/usr/share/doc/gcc-11-base/C++/README.C++
/usr/share/doc/gcc-11-base/C++/changelog.gz
/usr/share/lintian
/usr/share/lintian/overrides
/usr/share/lintian/overrides/g++-11
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/x86_64-linux-gnu-g++-11.1.gz
/usr/bin/g++-11
/usr/share/doc/g++-11
/usr/share/man/man1/g++-11.1.gz
---------- COMPELTED DPKG ----------
```

**Comment by awalsh128** [comment]
Please use https://github.com/awalsh128/cache-apt-pkgs-action/releases/tag/v1.0.3 as it contains fixes for caching. Let me know if it is able to resolve your issue.

**Comment by awalsh128** [comment]
Going to close this out since this fix has been confirmed in #21
